src/index.js

```diff
@@ -1,2 +1,24 @@
-// import _ from 'lodash'
 import './style.css';
+import newgame from './newgame.js';
+import saveuserAndScores from './save.js';
+import getdata from './getdata.js';
+const submit = document.getElementById('submit');
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+
+document.addEventListener('DOMContentLoaded', function () {
+  getdata();
+});
```

Comment:
- [ ] Would you refactor this and use arrow function?
```suggestion
document.addEventListener('DOMContentLoaded', () => getdata());
```